Adds the 2022 Rules Section for Behaviour (#229)
NUbots/NUbook: src/book/02-system/02-subsystems/06-behaviour.mdx

ScriptTuner also allows for "locking" and "unlocking" servos on the robot. When a servo is "locked", its position can only be changed in ScriptTuner. When it is "unlocked", the servo can be moved physically on the robot, which will update the position value in the script. Locking a servo involves emitting a ServoCommand for the current position of the servo. Unlocking the servo involves emitting a ServoCommand with zero torque and gain.

To learn how to use ScriptTuner, read the [ScriptTuner guide](/guides/main/tuning-and-running-scripts#script-tuner).

<details>
<summary>Play-States and Considerations (RoboCup 2022)</summary>

> Based on the rules as specified in the [RoboCup Humanoid Rules](https://github.com/RoboCup-Humanoid-TC/Rules) there are some states in the game such as when a penalty occurs in favor of the opposing team which recieves either a direct or indirect free kick, where the robots need to then act in a particular way such as standing still, or moving out of the way for the kick for the duration of the kick.

Reviewer: ```suggestion
Based on the specifications in the [RoboCup Humanoid Rules](https://github.com/RoboCup-Humanoid-TC/Rules) there are some states in the game where the robots need to then act differently to their regular behaviour, for example if the opposing team has been granted a free kick,  we might need to stand still, or move out of the way for the duration of the kick.
```

> [In the revision of the rules as per March 24th 2022:](http://humanoid.robocup.org/wp-content/uploads/RC-HL-2022-Rules.pdf)

Reviewer: To match the convention of dates elsewhere in nubook
```suggestion
[In the revision of the rules as per 24th of March 2022:](http://humanoid.robocup.org/wp-content/uploads/RC-HL-2022-Rules.pdf)
```

> ### _All free kicks are taken from the place where the offence occurred, except:_

Reviewer: ```suggestion
### All free kicks are taken from the place where the offence occurred, except:
```

> ### All free kicks are taken from the place where the offence occurred, except:

Reviewer 2: I don't think this should be a heading

> - _Indirect free kicks to the attacking team for an offence inside the opponents' penalty area are
>   taken from the nearest point on the penalty area line which runs parallel to the goal line._

Reviewer: ```suggestion
- Indirect free kicks to the attacking team for an offence inside the opponent's penalty area, which are taken from the nearest point on the penalty area line which runs parallel to the goal line.
```

> - _In the physical competition, free kicks to the defending team in their goal area may be taken
>   from anywhere in that area_

Reviewer: ```suggestion
- In the physical competition, where free kicks to the defending team in their goal area may be taken from anywhere in that area
```

> ### _Until the ball is in play all opponents must remain:_

Reviewer: ```suggestion
### Until the ball is in play all opponents must remain:
```

> - _0.75m for KidSize and 1.5m for AdultSize from the ball until it is in play, unless they are on their own goal line between the goalposts_

Reviewer: ```suggestion
- 0.75m for KidSize and 1.5m for AdultSize from the ball until it is in play, unless they are on their own goal line between the goalposts
```

> - _Outside the penalty area for free kicks inside the opponents' penalty area_

Reviewer: ```suggestion
- Outside the penalty area for free kicks inside the opponent's penalty area
```
